### PR TITLE
Category name order by ascending

### PR DESCRIPTION
**Pull request by SadikurSadik**
fixes: [1238](https://github.com/weDevsOfficial/dokan/issues/1238)

### CODE REVIEW

File: templates/products/new-product-single.php [hunk 1]
```diff
@@ -274,6 +274,7 @@ class="vendor-earning simple-product"
                                                 'name'             => 'product_cat',
                                                 'id'               => 'product_cat',
                                                 'taxonomy'         => 'product_cat',
+                                                'orderby'          => 'name',
```
Comment (alamgircsebd) [inline templates/products/new-product-single.php:277]: Please add a filter hook here, so anyone can overwrite the orderby value

Comment (alamgircsebd) [reply templates/products/new-product-single.php:277]: Also, make sure you have added the order by value on the popup area

**Comment by FaraziF** [comment]
The issue has not been resolved yet

File: templates/products/tmpl-add-product-popup.php [hunk 2]
```diff
@@ -103,13 +103,14 @@
...
                                 'id'               => 'product_cat',
                                 'taxonomy'         => 'product_cat',
+                                'orderby'          => 'name',
                                 'title_li'         => '',
                                 'class'            => 'product_cat dokan-form-control dokan-select2',
                                 'exclude'          => '',
                                 'selected'         => $product_cat,
                             );
 
-                            wp_dropdown_categories( apply_filters( 'dokan_product_cat_dropdown_args', $category_args ) );
```
[the remark this answers is not in the document]
Comment (SadikurSadik) [reply templates/products/tmpl-add-product-popup.php]: For testing purpose i was removed the filter but forgot to revoke. I pushed one commit for this changes.

**Comment by FaraziF** [comment]
When you select multiple product categories, then ascending order is not working.

Screenshot 2021-06-30 at 10 21 17 AM

**Comment by nurul-umbhiya** [comment]
@SadikurSadik what is the status of this pr?

**Comment by SadikurSadik** [comment]
> When you select multiple product categories, then ascending order is not working.
> 
> Screenshot 2021-06-30 at 10 21 17 AM

Fixed

**Review by nurul-umbhiya — APPROVED** [review]

**Pull request merged**